index.test.js

```diff
@@ -0,0 +1,47 @@
+import { addString, toNumber, filterOrange } from "./index";
+
+//question01
+describe("addString", () => {
+  test('adds a + b to equal ab', () => {
+    expect(addString('a', 'b')).toBe('ab');
+  });
+
+  test('adds a + 1 to equal a1', () => {
+    expect(addString('a', '1')).toBe('a1');
+  });
+
+  test('adds 1 + 1 to equal -1', () => {
+    expect(addString(1, 1)).toBe(-1);
+  });
+})
+
+
+//question02
+describe("toNumber", () => {
+  test('Convert the string to an integer to equal 1234', () => {
+    expect(toNumber('1234')).toBe(1234);
+  });
+
+  test('Convert the string to an integer to equal 1234', () => {
+    expect(toNumber('  1234TEST')).toBe(1234);
+  });
+
+  test('Convert the string to an integer to equal NaN', () => {
+    expect(toNumber('TEST1234')).toBe(NaN);
+  });
+})
+
+
+//question03
+describe("filterOrange", () => {
+  test('the fruit list has orange on it', () => {
+    const fruit = ["orange", "apple", "peach"];
+    expect(filterOrange(fruit)).toContain('orange');
+  });
+
+  test('the fruit list has orange on it', () => {
+    const fruit = ["orange", "apple", "peach"];
+    expect(filterOrange(fruit)).toEqual(['orange']);
+  });
```

Comment:
Add test that not include "orange" in fruit

Reply:
Thank you for the review!
I added the tests I came up with. 
latest commit: https://github.com/csakyo/JEST/pull/1/commits/283faf63b64715f0e05bbdac23c17b89cbb838ca